examples/datasource-basic/src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -17,19 +17,25 @@ export function ConfigEditor(props: EditorProps): ReactElement {
     <>
       <FieldSet label="General">
         <InlineField label="Time Field" tooltip="Default time field used when interpolate the $__timeFilter()">
-          <Input onChange={onTimeFieldChange} placeholder="time" value={jsonData?.defaultTimeField ?? ''} />
+          <Input
+            onChange={onTimeFieldChange}
+            placeholder="time"
+            aria-label="Macro default time field"
+            value={jsonData?.defaultTimeField ?? ''}
+          />
         </InlineField>
       </FieldSet>
 
       <FieldSet label="API Settings">
         <SecretFormField
           tooltip="API Key used to make calls to your data source"
-          isConfigured={Boolean(secureJsonData?.apiKey)}
+          isConfigured={Boolean(secureJsonFields['apiKey'])}
           value={secureJsonData?.apiKey || ''}
           label="API Key"
           placeholder="secure json field (backend only)"
           labelWidth={6}
           inputWidth={20}
+          aria-label="Secret api key"
```

Comment:
Super nit: we're trying to move things from aria-labels to data-testids. Unless the aria-label makes sense here from an a11y perspective. Also, instead of using this hardcoded string you should be using the same selector that you're using in the e2e tests.

Reply:
Great feedback! I started with that but when I used the `e2e.getSelectors` function it seems like it is expecting `aria-labels`.